unfoldingWord/translationCore
# MyProjects Actions/Reducers

#### This pull request addresses:

Adds Actions and Reducers required to populate the MyProjects section of Home Screen.
This moves all of the logic out of components into the action, including access time.

#### How to test this pull request:

Navigate to the Projects Management on the home screen and it should populate.
No actions work yet... that is next PR


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2026)
